test/JDBC/upgrade/16_8/schedule

```diff
@@ -598,4 +598,5 @@ test_conv_money_to_varchar
 fixeddecimal_modulo
 BABEL-NUMERIC_EXPR
 test_conv_float_to_varchar_char
+object_ownership-before-16_8-or-17_4
```

Comment:
Why are we only adding this to 16_8 schedule file? Shouldn't we add for few other previous versions as well?